kupfer.py

```diff
@@ -19,4 +20,7 @@
 
 if __name__ == '__main__':
     from kupfer import main
+#    from os import chdir
+#    chdir("/home/chaitanya/Documents/PROJECTS/kupfer/")
+#    chdir is required if kupfer script added in OS starup (start on boot). Otherwise all the applications are not indexed.
```

Comment:
Report a bug for that, be specific about what correctly indexed means (expected and actual behaviour). Linux is very diverse so there are a lot of cases.